docs/program/apis/sessions.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Sessions Management with Viam's Client SDKs"
+linkTitle: "Session Management API"
+weight: 20
+type: "docs"
+description: "How to use the Sessions Management API with Viam's Client SDKs."
+tags: ["client", "sdk", "viam-server", "networking", "apis", "robot api", "session", "sessions", "session management"]
+---
+
+The Session Management API provides support for robot session management.
+
+A Session allows a **client** to express that it is actively connected or authenticated to `viam-server` on your robot and
+supports stopping actuating components when the connection is no longer active.
+
+{{< alert title="Info" color="info" >}}
+
+### Purpose of Session Management
+
+When controlling a robot or fleet with the Viam platform, you want some kind of protocol or system-wide means to understand the presence of the clients that are talking and authenticated to each robot's `viam-server` through Viam's various [resource APIs](/program/apis/).
+
+The presence of these clients at one point in time can be referred to as a "session".
```

Comment:
Adding! I think it's hard to get conceptually and hard to phrase. Using _is_ is definitely better, though

Reply:
Yeah it's tricky, and I Googled it for a bit before making that comment^^ to see if there are other ways of conceptualizing and/or phrasing it. Everyone seems to have a hard time putting it into words :D But I think it can be improved. In this case, is it indeed more of a period of time?